README.md

```diff
@@ -69,4 +69,17 @@ See the following minimal examples of the AsyncAPI Converter usage in the termin
 
 ### As a package
 
-To see examples how to use the AsyncAPI Converter as a package, go to the [README.md](./examples/README.md). 
\ No newline at end of file
+To see examples how to use the AsyncAPI Converter as a package, go to the [README.md](./examples/README.md).
```

Comment:
```suggestion
To see examples of how to use the AsyncAPI Converter as a package, go to the [README.md](./examples/README.md).
```

Reply:
In the [README.md](https://github.com/m00g3n/converter-go/blob/readme-update/examples/README.md), please also add `of` in the same place as in the above sentence:
`This directory contains examples of how to use the AsyncAPI Converter as a package.`